README.md

```diff
@@ -0,0 +1,11 @@
+# AppFormer.js
+
+
+Getting started
+--
+#### 1. Building
+
+1. `cd packages/core`
+2. `yarn run init && yarn run build`
```

Comment:
Hangs forever on my machine
```bash
yarn run v1.12.3
$ yarn -f --registry http://localhost:4873 --no-lockfile
[1/4] Resolving packages...
⢀ @types/react@^16.4.18
```
yarn: 1.12.3
npm: 6.4.1
node: v8.12.0

Reply:
It will be necessary to have `verdaccio` installed in the future. The `core` package doesn't need it, so I'm removing it from the `init` script.

Reply:
Works for me, however I see these warnings about circular dependencies:

```WARNING in Circular dependency detected:
src/java-wrappers/JavaByte.ts -> src/java-wrappers/JavaNumber.ts -> src/java-wrappers/JavaByte.ts

WARNING in Circular dependency detected:
src/java-wrappers/JavaDouble.ts -> src/java-wrappers/JavaNumber.ts -> src/java-wrappers/JavaDouble.ts

WARNING in Circular dependency detected:
src/java-wrappers/JavaFloat.ts -> src/java-wrappers/JavaNumber.ts -> src/java-wrappers/JavaFloat.ts

WARNING in Circular dependency detected:
src/java-wrappers/JavaInteger.ts -> src/java-wrappers/JavaNumber.ts -> src/java-wrappers/JavaInteger.ts

WARNING in Circular dependency detected:
src/java-wrappers/JavaLong.ts -> src/java-wrappers/JavaNumber.ts -> src/java-wrappers/JavaLong.ts

WARNING in Circular dependency detected:
src/java-wrappers/JavaNumber.ts -> src/java-wrappers/JavaDouble.ts -> src/java-wrappers/JavaNumber.ts

WARNING in Circular dependency detected:
src/java-wrappers/JavaShort.ts -> src/java-wrappers/JavaNumber.ts -> src/java-wrappers/JavaShort.ts

WARNING in Circular dependency detected:
src/marshalling/MarshallerProvider.ts -> src/marshalling/marshallers/JavaHashMapMarshaller.ts -> src/marshalling/marshallers/util/GenericsTypeMarshallingUtils.ts -> src/marshalling/MarshallerProvider.ts

WARNING in Circular dependency detected:
src/marshalling/marshallers/DefaultMarshaller.ts -> src/marshalling/MarshallerProvider.ts -> src/marshalling/marshallers/DefaultMarshaller.ts

WARNING in Circular dependency detected:
src/marshalling/marshallers/JavaCollectionMarshaller.ts -> src/marshalling/marshallers/util/GenericsTypeMarshallingUtils.ts -> src/marshalling/MarshallerProvider.ts -> src/marshalling/marshallers/JavaCollectionMarshaller.ts

WARNING in Circular dependency detected:
src/marshalling/marshallers/JavaHashMapMarshaller.ts -> src/marshalling/marshallers/util/GenericsTypeMarshallingUtils.ts -> src/marshalling/MarshallerProvider.ts -> src/marshalling/marshallers/JavaHashMapMarshaller.ts

WARNING in Circular dependency detected:
src/marshalling/marshallers/JavaOptionalMarshaller.ts -> src/marshalling/marshallers/util/GenericsTypeMarshallingUtils.ts -> src/marshalling/MarshallerProvider.ts -> src/marshalling/marshallers/JavaOptionalMarshaller.ts

WARNING in Circular dependency detected:
src/marshalling/marshallers/util/GenericsTypeMarshallingUtils.ts -> src/marshalling/MarshallerProvider.ts -> src/marshalling/marshallers/JavaHashMapMarshaller.ts -> src/marshalling/marshallers/util/GenericsTypeMarshallingUtils.ts```